src/components/Navbar/index.tsx

```diff
@@ -229,24 +215,6 @@ export class NavbarBase extends React.Component<Props, State> {
           </div>
         </Navbar.Brand>
         <Navbar.Text className={styles.text}>
-          {currentVersionId ? (
-            <a
-              className={styles.legacyLink}
-              href={
-                currentBaseVersionId
-                  ? `${baseUrlToLegacy}/compare-redirect/${currentVersionId}...${currentBaseVersionId}/${legacyQuerystring(
-                      path,
-                    )}`
-                  : `${baseUrlToLegacy}/browse-redirect/${currentVersionId}/${legacyQuerystring(
-                      path,
-                    )}`
-              }
-              rel="noopener noreferrer"
-              target="_blank"
-            >
-              {gettext('Legacy Viewer')}
-            </a>
-          ) : null}
```

Comment:
I think we can also remove the `withRouter()` HOC. I don't see it used anywhere anymore.